File: Algorithm/2/isPrime.py
import math
def isPrime(n):
    if n == 0 or n == 1:
        return False
    if n == 2:
        return True
    if n < 0:
        return False
    if n % 2 == 0:
        return False
    for i in range(3,int(math.sqrt(n))+1, 2):
        if n%i == 0:
            return False
    return True

print(isPrime(2))

File: Algorithm/2/isPrime.cpp
#include <iostream>
using namespace std;
bool isPrime(int n){
  for(int x = 2; x*x <= n; x++)
    if(n%x == 0)
      return false;
  return true;
}

bool isPrime(int n){
  if(n == 0 || n == 1) return false;
  if(n == 2) return true;
  if(n < 0) return false;
  if(n%2 == 0) return false
  for(int x = 3; x*x <= n; x += 2)
    if(n%x == 0)
      return false;
  return true;
}
